src/components/image/index.tsx

```diff
@@ -112,6 +117,21 @@ export function Image({
     }
   }, [data, fetchingFileInProgress, imgError]);
 
+  const handleDelete = async () => {
+    if (!propThing || !solidDataset || !propProperty || !value) return;
+    try {
+      const updatedThing = removeUrl(propThing, propProperty, value as string);
+      const updatedDataset = setThing(solidDataset, updatedThing);
+      const datasetSourceUrl = getSourceUrl(solidDataset);
+      if (!datasetSourceUrl) return;
+      await saveSolidDatasetAt(datasetSourceUrl, updatedDataset, {
+        fetch,
+      });
```

Comment:
Shouldn't this only happen when `autosave` is set ? Or is it by design that when deleting an image you're automatically deleting it from the remote storage ?

Reply:
Good question. I am not sure but I have added a check for autosave since it is what we do with other operations as well.